.csscomb.json

```diff
@@ -1,24 +1,316 @@
 {
-    "remove-empty-rulesets": true,
     "always-semicolon": true,
-    "color-case": "lower",
     "block-indent": "    ",
+    "color-case": "lower",
     "color-shorthand": true,
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": false,
+    "leading-zero": true,
     "quotes": "double",
+    "tab-size": true,
+    "remove-empty-rulesets": true,
+    "strip-spaces": true,
+    "unitless-zero": true,
+    "vendor-prefix-align": true,
+    "space-between-declarations": "\n",
     "space-before-colon": "",
     "space-after-colon": " ",
     "space-before-combinator": " ",
     "space-after-combinator": " ",
-    "space-between-declarations": " ",
     "space-before-opening-brace": " ",
     "space-after-opening-brace": "\n",
-    "space-after-selector-delimiter": " ",
     "space-before-selector-delimiter": "",
+    "space-after-selector-delimiter": "\n",
     "space-before-closing-brace": "\n",
-    "strip-spaces": true,
-    "tab-size": true,
-    "unitless-zero": true
+    "sort-order": [
+        "$variable",
+        "$extend",
+        "$include",
+        "content",
+        "display",
+        "position",
+        "visibility",
+        "float",
+        "clear",
+        "overflow",
+        "overflow-x",
+        "overflow-y",
+        "-ms-overflow-x",
+        "-ms-overflow-y",
+        "clip",
+        "zoom",
+        "flex-direction",
+        "flex-order",
+        "flex-pack",
+        "flex-align",
+        "z-index",
+        "top",
+        "right",
+        "bottom",
+        "left",
+        "-webkit-box-sizing",
+        "-moz-box-sizing",
+        "box-sizing",
+        "width",
+        "min-width",
+        "max-width",
+        "height",
+        "min-height",
+        "max-height",
+        "margin",
+        "margin-top",
+        "margin-right",
+        "margin-bottom",
+        "margin-left",
+        "padding",
+        "padding-top",
+        "padding-right",
+        "padding-bottom",
+        "padding-left",
```

Comment:
`border` properties should then follow `padding margin` etc, as they form the box model